hackathon/forms.py

```diff
@@ -71,10 +71,9 @@ class HackathonForm(forms.ModelForm):
         required=True,
         widget=forms.Select(choices=STATUS_TYPES_CHOICES),
     )
-    judging_status = forms.CharField(
-        label="Judging Status",
+    teamsize = forms.IntegerField(
```

Comment:
Probably just me being pedantic, but shouldn't the naming convention match the others?
`team_size` instead of `teamsize`?

Reply:
Updated to `team_size`